ocs_ci/framework/main.py

```diff
@@ -36,17 +48,24 @@ def init_ocsci_conf(arguments=None):
     """
     if not arguments:
         return
-    custom_config = get_param('--ocsci-conf', arguments)
+    custom_config = get_param('--ocsci-conf', arguments, repeatable=True)
     cluster_config = get_param('--cluster-conf', arguments)
-    if custom_config:
-        with open(os.path.expanduser(custom_config)) as file_stream:
+    for config_file in custom_config:
+        with open(
+            os.path.abspath(os.path.expanduser(config_file))
+        ) as file_stream:
             custom_config_data = yaml.safe_load(file_stream)
             framework.config.update(custom_config_data)
     if cluster_config:
         with open(os.path.expanduser(cluster_config)) as file_stream:
             cluster_config_data = yaml.safe_load(file_stream)
             framework.config.update(cluster_config_data)
     framework.config.RUN['run_id'] = int(time.time())
+    bin_dir = framework.config.RUN.get('bin_dir')
```

Comment:
since we are already loading bin-dir in line 81, can we append bin_dir  in the same place ( after line 82 ).

Reply:
Moved everything here to be on one place